### PR TITLE
refact: use async mode for computing task & fix some bugs

### PR DESCRIPTION
**Pull request by imbajin**
problems list:
1. kube conf only support absolute path (fixed)
2. Computer Task belongs to SysJob ( fixed)
3. Computer Task type is wrong,  isComputerTask return false (fixed)
4. cancel API return success but not work indeed  (**TODO**)

And due to update hugegraph-computer (k8s api to async method), we need update computer dependency first then the ci could run

### CODE REVIEW

File: hugegraph-api/src/main/java/com/baidu/hugegraph/api/job/ComputerDisAPI.java [hunk 1]
```diff
@@ -76,10 +78,10 @@
     public Map<String, Object> post(@Context GraphManager manager,
                                     @PathParam("graph") String graph,
                                     JsonTask jsonTask) {
-        LOG.debug("Schedule computer dis job: {}", jsonTask);
-        E.checkArgument(K8sDriverProxy.isK8sApiEnabled() == true,
-                        "The k8s api is not enable.");
         checkCreatingBody(jsonTask);
+        E.checkArgument(K8sDriverProxy.isK8sApiEnabled(),
+                        "The k8s api is not enable.");
+        LOG.info("Schedule computer dis job: {}, graph is {}", jsonTask, graph);
```
Comment (zhoney) [inline hugegraph-api/src/main/java/com/baidu/hugegraph/api/job/ComputerDisAPI.java:84]: keep debug？

File: hugegraph-core/src/main/java/com/baidu/hugegraph/k8s/K8sDriverProxy.java [hunk 2]
```diff
@@ -111,15 +121,22 @@ protected void initConfig(String partitionsCount,
...
-    public KubernetesDriver getKubernetesDriver() {
-        return this.driver;
+    public KubernetesDriver getK8sDriver() {
+        return driver;
     }
 
     public void close() {
-        this.driver.close();
+        // TODO: Comment now & delete this method after ensuring it
+        //driver.close();
```
Comment (zhoney) [inline hugegraph-core/src/main/java/com/baidu/hugegraph/k8s/K8sDriverProxy.java:140]: when to close？maybe close when HugeGraphServer shutdown

**Review by zhoney — CHANGES_REQUESTED** [review]

**Comment by zhoney** [comment]
compile error:
```
Error:  Failed to execute goal on project hugegraph-core: Could not resolve dependencies for project com.baidu.hugegraph:hugegraph-core:jar:0.12.0: com.baidu.hugegraph:computer-k8s:jar:0.1.1 was not found in https://repo.maven.apache.org/maven2 during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of central has elapsed or updates are forced -> [Help 1]
```

File: hugegraph-core/src/main/java/com/baidu/hugegraph/k8s/K8sDriverProxy.java (hunk 2 above)
Comment (imbajin) [reply hugegraph-core/src/main/java/com/baidu/hugegraph/k8s/K8sDriverProxy.java:140]: no need to close until server shutdown (remove this method after ensure it)

File: hugegraph-api/src/main/java/com/baidu/hugegraph/api/job/ComputerDisAPI.java (hunk 1 above)
Comment (imbajin) [reply hugegraph-api/src/main/java/com/baidu/hugegraph/api/job/ComputerDisAPI.java:84]: mavbe save computer job log to server.log is better (for user friendly)? (only include new job & delete job now)

set to debug is also OK

**Comment by imbajin** [comment]
> compile error:
> 
> ```
> Error:  Failed to execute goal on project hugegraph-core: Could not resolve dependencies for project com.baidu.hugegraph:hugegraph-core:jar:0.12.0: com.baidu.hugegraph:computer-k8s:jar:0.1.1 was not found in https://repo.maven.apache.org/maven2 during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of central has elapsed or updates are forced -> [Help 1]
> ```

wait for computer `v0.1.1` version, now ci is OK

**Review by zhoney — APPROVED** [review]

**Pull request merged**